uuids.nim

```diff
@@ -20,7 +20,7 @@ template toHex(s: string, start: Natural,
     if n == 0 and x < 0: n = -1
 
 proc uuidsParseHexInt(s: string, maxLen: int): int64 =
-  if s.isNil or s.len == 0:
+  if s.len == 0:
```

Comment:
This changes behaviour for the older version of Nim - I'd prefer to keep it. You can add compile-time version check using `NimMajor`, `NimMinor`, `NimPatch` constants.

Reply:
This change is needed to compile with current devel anyway.
0.19 is a breaking change in that regard.
More over in 0.18 I'm quite sure `string.len` returned 0 on a `nil` `string`, since it was compiler magic.

Reply:
Yeah just checked and older version did return `0` for as a length of a `nil` string, so this is fine.